src/hiv-labs/hiv-lab-summary.component.tsx

```diff
@@ -30,11 +36,38 @@ export default function HivLabSummary(props: any) {
           orderByObsDate: Sort.DESC,
           top: 1
         });
+        setOrdersParams({
+          patientUuid: uuid,
+          conceptUuids: [VIRAL_LOAD_COUNT_CONCEPT, CD4_COUNT_CONCEPT],
+          orderByDateActivated: Sort.DESC,
+          top: 1
+        });
       });
     }
     return () => sub && sub.unsubscribe();
   }, []);
 
+  useEffect(() => {
+    if (obsByConcept && ordersByConcept) {
+      let pending: Array<Order> = [];
+      [CD4_COUNT_CONCEPT, VIRAL_LOAD_COUNT_CONCEPT].forEach(conceptUuid => {
+        if (ordersByConcept[conceptUuid].length > 0) {
+          if (obsByConcept[conceptUuid].length > 0) {
```

Comment:
```suggestion
        if (ordersByConcept[conceptUuid].length > 0 && obsByConcept[conceptUuid].length > 0)
```

Reply:
There can be an order, but not an obs results. By adding your change, it means we will miss this case.

Reply:
:+1: let's merge